docs/_publish/partner-program/partner-program.md

```diff
@@ -9,32 +9,32 @@ redirect_from:
 
 # Partner Program 
 
-> **The Partner Program is only eligible for public integrations.**
+> **The Partner Program is only available to public integrations.**
 
 The [Zapier Partner Program](https://zapier.com/developer-platform/partner-program) is a program for Zapier’s [6,000+ integration partners](https://zapier.com/apps). It is designed to give all partners a clear path to success for their integrations and reward them with benefits along the way.
 
-As you get more users and maintain a healthy integration, your integration accesses more benefits across our tiered system. We have 4 levels:
+As your integration gains more active users, and maintains a healthy health score, your integration will tier up in our Partner Program. Zapier's Partner Program consistes for 4 tiers:
```

Comment:
healthy health score sounds strange 😊 - should that be high health score? satisfactory health score?

Reply:
Haha yeah, it does sound weird. But that is technically (?) the designation in dev platform and how we refer to it in tickets with partners. eg. https://cdn.zappy.app/18ccd475d668beec8da9dd1526a6d3ff.png

Open to your suggestions! I just want the verbiage to match what's shown in dev platform